site/pages/component/gc-features-en.hbs

```diff
@@ -268,6 +269,67 @@
 	&lt;/div&gt;
 &lt;/section&gt;</code></pre>
 
+<h5>Cohabit design pattern</h5>
+<p>To be used in this context:</p>
+<pre><code>&lt;div class="row row-eqht mrgn-tp-xl"&gt;
+	&lt;div class="col-md-8"&gt;
+		&lt;section class="gc-features"&gt;
+			&lt;!-- gc-feature here --&gt;
+		&lt;/section&gt;
+	&lt;/div&gt;
+	&lt;div class="col-md-4"&gt;
+		&lt;!-- Cohabit component here. For example the Follow us component --&gt;
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h6>One feature tile - Cohabit design pattern</h6>
+<pre><code>&lt;div class="row row-eqht mrgn-tp-xl"&gt;
```

Comment:
We should try to use the pure CSS equal height instead of the JS plugin at this point.